src/dmd/backend/outbuf.d

```diff
@@ -38,6 +40,7 @@ struct Outbuffer
 
     //~this() { dtor(); }
 
+    @trusted
```

Comment:
What about putting such qualifiers after the function header to save space?

Reply:
I wanted to make it easy to add/remove/change it without having the function header in the "different" list.